Source/WebKit/UIProcess/API/glib/WebKitPrivate.cpp

```diff
@@ -150,6 +154,23 @@ unsigned toWebKitError(unsigned webCoreError)
     }
 }
 
+#if ENABLE(WK_WEB_EXTENSIONS)
+unsigned toWebKitWebExtensionMatchPatternError(unsigned apiError)
+{
+    switch (apiError) {
+    case static_cast<int>(WebKit::WebExtensionMatchPattern::Error::InvalidScheme):
+        return WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR_INVALID_SCHEME;
+    case static_cast<int>(WebKit::WebExtensionMatchPattern::Error::InvalidHost):
+        return WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR_INVALID_HOST;
+    case static_cast<int>(WebKit::WebExtensionMatchPattern::Error::InvalidPath):
+        return WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR_INVALID_PATH;
+    case static_cast<int>(WebKit::WebExtensionMatchPattern::Error::Unknown):
```

Comment:
Nite: might be nicer to cast apiError to `WebKit::WebExtensionMatchPattern::Error` once instead of casting to int four times.